src/main/scala/uk/gov/nationalarchives/DynamoWriteUtils.scala

```diff
@@ -57,8 +57,8 @@ object DynamoWriteUtils {
           checksumSha256 -> DynamoValue.fromString(fileDynamoTable.checksumSha256),
           fileExtension -> DynamoValue.fromString(fileDynamoTable.fileExtension),
           representationType -> DynamoValue.fromString(fileDynamoTable.representationType match {
-            case DynamoFormatters.RepresentationTypePreservation => "Preservation"
-            case DynamoFormatters.RepresentationTypeAccess       => "Access"
+            case DynamoFormatters.PreservationRepresentationType => "Preservation"
```

Comment:
Now that you've added a `toString`, Do you think it makes sense to do `DynamoFormatters.PreservationRepresentationType.toString` instead of `"Preservation"`? or 

```
case presResType: DynamoFormatters.PreservationRepresentationType => presResType.ToString // I forgot whether it's ":" or "@"
```

Same question for `"Access"`, below and line 71 and 72 in the DynamoReadUtils file

Reply:
We don't need this match statement in the WriteUtils at all any more. It can just be `DynamoValue.fromString(fileDynamoTable.representationType.toString)`

The static strings in the ReadUtils need to stay as is, the compiler complains if I try to use the toString method.